bedrock/firefox/templates/firefox/new/break-free/scene1.html

```diff
@@ -4,6 +4,12 @@
 
 {% extends "firefox/new/break-free/base.html" %}
 
+{% block optimizely %}
+  {% if switch('firefox-new-variant-optimizely') %}
```

Comment:
Adding a new switch name here so we don't have to load Optimizely on the canonical `/firefox/new/` while these experiments are running.